DOCS-16527 replica set to sharded cluster page (#363)
mongodb/docs-cluster-to-cluster-sync: source/includes/limitations-rs-to-sharded.rst

- ``mongosync`` allows users to rename collections that the
  ``sharding.shardingEntries`` option includes during sync. To see renaming 
  limitations, see :ref:`Renaming

>   During Sync <rename-during-sync>`.

Author: Thanks for the review! Fixing now.

>   ``sharding.shardingEntries`` option includes during sync. To see limitations 
>   on renaming collections while ``mongosync`` is running, see :ref:`Renaming
>   During Sync <rename-during-sync>`.

Reviewer: [suggestion] for minimalism

```suggestion
  ``sharding.shardingEntries`` option includes during sync. To see 
  renaming limitations, see :ref:`Renaming
  During Sync <rename-during-sync>`.
```

Author: Thanks for the suggestion, changing now!

> - When using the ``sharding.createSupportingIndexes`` option to create

Reviewer: [suggestion] for more [user centric language](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/#write-to-the-user)

```suggestion
- If you use the ``sharding.createSupportingIndexes`` option to create
```

Author: Thanks for the suggestion, changing now!

  indexes supporting shard keys on the destination cluster during
  sync, you cannot create these indexes afterwards on the source
  cluster.

>   The index must either exist before ``mongosync`` starts or be
>   created after the migration is complete and ``mongosync`` has
>   stopped.

Reviewer: [suggestion] for clarity and active voice

```suggestion
  If you want to create an index that supports shard keys, you must create the index before ``mongosync`` starts   or after the migration is complete and ``mongosync`` has
  stopped.
```

>   ``mongosync`` has stopped.

Author: I like these changes, editing now!

- ``mongosync`` allows users to rename collections that the
  ``sharding.shardingEntries`` option includes during sync. For details, see 

>   :ref:`Renaming During Sync <rename-during-sync>`.

Author: This was already on this page, but I think the point is that users should look at "Renaming During Sync" for details about limitations when renaming during sync. I made a change to it, please let me know if it is clearer!

- If you use the ``sharding.createSupportingIndexes`` option, the indexes to
  support the shard keys are automatically created on the destination cluster

>   during the sync. 

Author: I am going to keep it as two separate limitations, but will reword it.
